wizeline/remix-project-lab
# Feat/229/improve view of applicants postullated in projects

# What does this PR do?
Improves the way applicants are displayed in projects and extracts the profile picture from linkdin



# Where should the reviewer start?
app/routes/applicants/$applicantId/index.tsx
app/routes/projects/$projectId/index.tsx



# How should this be manually tested?
1. Log in to linkdin and fill out the form.
2. Apply to any project
3. login with google
4. Verify that the applicant's photo is displayed in /applicants.
5. Verify that the project you applied to shows the applicant correctly.



# Any background context you want to provide?



# What are the relevant tickets?
#229 



# Screenshots
<img width="1728" alt="Captura de pantalla 2023-11-07 a la(s) 8 58 40 a m" src="https://github.com/wizeline/remix-project-lab/assets/58340818/37ab9c62-49dc-4347-a09f-c9e8b11d1811">

<img width="1728" alt="Captura de pantalla 2023-11-07 a la(s) 8 58 48 a m" src="https://github.com/wizeline/remix-project-lab/assets/58340818/21c3bb08-7a9b-4442-ae5d-a631ac084fc6">




# Questions



# Checklist
